### ISSUE TITLE
Set up Dependabot?

### ISSUE THREAD

**Issue by theodore-s-beers** [M-L-D-H/Closing-The-Gap-In-Non-Latin-Script-Data#71]
NPM dependencies tend to move, and break, quickly. Dependabot can be set to check for updates on a weekly or monthly basis (or daily, though I don't recommend that). It also opens PRs, which can work well, depending on the CI setup.

**Comment by Jomula** [comment]
Good idea. I haven't done that before, so if you know how to set up one, feel free. Otherwise, I can also assign the issue to myself and work on it next time I find some time.

**Comment by theodore-s-beers** [comment]
You're going to have to do it, since it requires full admin privileges on the repo. But I can at least help by suggesting how to fill in the `dependabot.yml` file. (When you go to set up Dependabot, under Insights > Dependency graph, it'll give you a template, to which a few things will need to be added for it to work properly.)

**Comment by Jomula** [comment]
I've set up a dependabot with version updates, the `dependabot.yml` is just the basic one now, checking on `gui` branch just for it to be set up, yaml can be adjusted.

**Comment by theodore-s-beers** [comment]
You did everything right. I would only adjust the frequency to monthly, so that the PRs don't become too much of a hassle.
